hrjobcontract/js/src/job-contract/controllers/modal/modal-contract.js

```diff
@@ -216,6 +215,9 @@ define([
         }
       }
 
+      // Call to fetch updated Health Plan Types
+      fetchHealthPlanTypes();
```

Comment:
Could you wrap all the logic that happens immediately into a IIFE?
```js
// Init
(function init() {
  angular.forEach($scope.files, function(entityFiles, entityName) {
    $scope.filesTrash[entityName] = [];
  });

  $modalInstance.opened.then(function() {
    $rootScope.$broadcast('hrjc-loader-hide');
  });

  angular.forEach($scope.uploader, function(entity) {
    angular.forEach(entity, function(field) {
      field.onAfterAddingAll = function() {
        $scope.filesValidate();
      }
    });
  });

  fetchHealthPlanTypes();
}());
```

The overall code structure should be as following:
```js
// variable/properties definitions
var content = content || {};

$scope.entity = {};
$scope.action = action;
$scope.copy = copy;

// methods published on the $scope
$scope.cancel = function () {}
$scope.fileMoveToTrash = function () {}

// init
(function init() {

}());

// private functions
function convertToDateObject() {}
function contractEdit() {}
```